content/guides/nextjs/github-open-authoring/loading-github-content.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Loading Content from GitHub
+---
+
+Now that we have authentication and the _Working Repository_ set up, it's time to wire up some content to edit.
+
+Check out `pages/index.tsx`, right now the content for this page is statically written into the component. Let's create a data file to source this content from. Add a new directory in the root of your project called `content` (or `data`, whichever you prefer) and create a new file called `home.json`. We'll start small by just adding editing a title.
+
+**content/home.json**
+
+```json
+{
+  "title": "Give me your tots 🦙"
+}
+```
+
+Back in `pages/index.tsx`, we need to set up data fetching. We will use [_getStaticProps_](https://nextjs.org/docs/basic-features/data-fetching#getstaticprops-static-generation) to return different sets of data based on the "Preview" or "Edit Mode".
+
+**pages/index.tsx**
+
+```diff
+import Head from 'next/head'
+/**
+ * Import helpers and GetStaticProps type
+ */
++ import { getGithubPreviewProps, parseJson } from 'next-tinacms-github'
++ import { GetStaticProps } from 'next'
+
+- export default function Home() {
++ export default function Home({ file }) {
++  const data = file.data
```

Comment:
I love the formatting but the `diff` code blocks really mess with the flow. All the `+` signs end up getting copied into your source file. I wonder if there's a way around that 🤔

Reply:
I feel ya, it does a great job of visually communicating the steps but then implementing it feels clunky. open to suggestions!